apps/face-recognition.sh

```diff
@@ -90,7 +90,13 @@ fi
 # Install requirements
 # https://github.com/matiasdelellis/facerecognition/wiki/Installation#ubuntu-focal
 echo "deb https://repo.delellis.com.ar focal focal" > /etc/apt/sources.list.d/20-pdlib.list
-wget -qO - https://repo.delellis.com.ar/repo.gpg.key | sudo apt-key add -
+if ! curl -fsSL https://repo.delellis.com.ar/repo.gpg.key | sudo apt-key add -
```

Comment:
I was thinking the exact same thing actually, again. :D

But, I think maybe we should use the `curl_to_dir` function instead and improve that function for all the scripts with something like this.

Reply:
hm... can we do that in a follow up? 
It is honestly a bit out of scope for this PR and should better be done in a different PR, imo.